src/cascadia/TerminalControl/XamlLights.h

```diff
@@ -5,45 +5,115 @@
 
 #include "cppwinrt_utils.h"
 #include "VisualBellLight.g.h"
+#include "CursorLight.g.h"
+
+#define CREATE_XAML_LIGHT(name)                                                                                                                 \

```

Comment:
This macro is also weird - is this again because these static methods have to be defined per-class, and not just at some base class level? 

That would definitely be annoyingly un-ergonomic from an API design POV.